### ISSUE TITLE
Ability to Adjust Position of Alphabet

### ISSUE THREAD

**Issue by Cod3rQwerty** [8bitPit/Niagara-Issues#2834]
### Is your feature request related to a problem?

I feel it would be useful to be able to customize the height or positioning of the alphabet as I have an app that allows me to use my phone completely one-handed but the alphabet interferes with it so I would like to be able to move it up a little.

### Describe the solution you'd like

I would like to have an option to adjust the height of the alphabet. Perhaps in a sub-section in the "Look" section in settings. This would be best implemented (in my opinion) by having a slider to adjust the height of it.

### Feature Request Procedure

- [X] I have read the [knowledge base article on feature requests](https://help.niagaralauncher.app/article/44-feature-requests) and followed the steps mentioned

**Comment by contact-maximux** [comment]
perfect feature suggestion. When niagra focusing using favorite then just feather touch on side and app drawer is kinda off for me. I always wanted some solution to deminish behaviour of app drawer by accidently touching screen. Thanks for perfect suggestion

**Comment by contact-maximux** [comment]
one solution could be limit height and location of side edge for app drawer. I know niagra team has made tons of effort to develop this snakescroller app drawer. But believe me few people who pay niagra subscription do not want it. Or at least do not want it super sensitive to touch anywhere in right edge

**Comment by swiftwind501** [comment]
I agree; I would prefer if the alphabet trigger zone were strictly under the widget stack where it can adjust with the stack's position.

From my experience, the sensitivity is sometimes inconsistent; if it functioned just like iOS's scrolling alphabet for all lists and had a static trigger zone defined by the UI element, then I'd be happy. Meanwhile, other Android launchers treat the app list as a single scrolling page rather than one sectioned off by the first letter in each app's name, so Niagara is the only one that mimics iOS's design.
